src/main/scala/gitbucket/core/api/MergeAPullRequest.scala

```diff
@@ -0,0 +1,23 @@
+package gitbucket.core.api
+
+/**
+ * https://developer.github.com/v3/pulls/#merge-a-pull-request
```

Comment:
Hmm, this site is now seem to be outdated.
@takezoe Is it better to use https://docs.github.com/en/rest/reference/pulls#merge-a-pull-request-merge-button intead?

Reply:
Sorry, this is the right URL 😅 
https://docs.github.com/en/rest/reference/pulls#merge-a-pull-request

Reply:
OK. I replace with a new URL.

Reply:
@SIkebe Thank you for confirming!!